### ISSUE TITLE
jQuery 3 support

### ISSUE THREAD

**Issue by martindrzka** [woocommerce/selectWoo#16]
It seems that plugin doesn't work correctly with lastest version of jQuery. You can list items but screenreaders don't read selected one.

- https://woocommerce.github.io/selectWoo/accessibility-testing/samples/ajax.html (jQuery v1.11.2)
- http://drzka.cz/selectwoo/accessibility-testing/samples/ajax.html (jQuery v3.3.1)

Any chance for latest jQuery support in the near future?

**Comment by irbian** [comment]
Any news on this?
